src/aat/java/uk/gov/hmcts/reform/em/stitching/testutil/TestUtil.java

```diff
@@ -202,6 +204,41 @@ public String uploadDocX(String docName) {
         return newDocUrl;
     }
 
+    public BundleDTO getTestBundleWithDuplicateBundleDocuments() {
+        BundleDTO bundle = new BundleDTO();
+        bundle.setBundleTitle("Bundle Title");
+        bundle.setDescription("This is the description of the bundle: it is great.");
+        List<BundleDocumentDTO> docs = new ArrayList<>();
+        docs.add(getTestBundleDocument(uploadDocument(), "Document 1"));
+        docs.add(getTestBundleDocument(uploadDocument(), "Document 1"));
```

Comment:
technically these will have different IDs. You might want to do `uploadDocument()` once and store the result.